src/GameState/Board.tsx

```diff
@@ -1,48 +1,64 @@
-import { Stack, HStack, Box } from 'native-base';
+import { Box, View } from 'native-base';
 import React from 'react';
-import { BOARD_GREEN, BOARD_GREY } from './boardColors';
+import { ImageBackground } from 'react-native';
+import { BOARD_GREEN } from './boardColors';
+import { convertBoardToBoardCoordinate } from './convertBoardToBoardCoordinate';
 import { Marble } from './Marble';
 
 export const Board = (props: any) => {
   const board = props.board;
-  const boardSize = props.preview ? 4 : 12;
-  const marbleColorSize = props.preview ? 4 : 12;
-  const marbleEmptySize = props.preview ? 2 : 4;
+  const boardSize = props.preview ? '128px' : '364px';
+  const marbleColorSize = props.preview ? '16px' : '48px';
```

Comment:
We usually use constants. But don't change it now.

Reply:
It's done in PR #15